src/Fixer/Contrib/RandomApiMigrationFixer.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFunctionReferenceFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Vladimir Reznichenko <kalessil@gmail.com>
+ */
+final class RandomApiMigrationFixer extends AbstractFunctionReferenceFixer
+{
+    /**
+     * @var string[]
+     */
+    private static $replacements = array(
+        'rand' => 'mt_rand',
+        'srand' => 'mt_srand',
+        'getrandmax' => 'mt_getrandmax',
+    );
+
+    /**
+     * @param string[]|null $customReplacements
+     */
+    public function configure(array $customReplacements = null)
```

Comment:
I can add turning replacements individually off (by setting up replacement to null). Shall I do this?